kong/cmd/start.lua

```diff
@@ -18,7 +18,10 @@ local function execute(args)
   local dao = assert(DAOFactory.new(conf))
   xpcall(function()
     assert(prefix_handler.prepare_prefix(conf, args.nginx_conf))
-    assert(dao:run_migrations())
+    if args.run_migrations then
+      assert(dao:run_migrations())
+    end
+    assert(dao:are_migrations_uptodate())
```

Comment:
It is indeed fatal, since the `assert` will fail and return the error. The error is that migrations are not up to date, the `warn` just tells which migration is missing (it's a more verbose information). I could log it as `verbose` or `error`.